src/components/Game.jsx

```diff
@@ -1,34 +1,76 @@
 import React, { Component } from 'react'
-import {shuffleBoard} from './Common/shuffleBoard'
+import {shuffleBoard, copyBoard} from './Common/shuffleBoard'
 import Board from './Board'
 import ScoreBox from './ScoreBox'
 import CurrentWord from './CurrentWord'
 import './Game.css'
 class Game extends Component {
 
-  constructor() {
-    super()
-    this.board = shuffleBoard()
-    this.handleClick = this.handleClick.bind(this)
-    this.handleSubmit = this.handleSubmit.bind(this)
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.initBoard = shuffleBoard();
+    this.surroundingCells = [
+      {r:-1, c:-1},
+      {r:-1, c:0},
+      {r:-1, c:1},
+      {r:0, c:-1},
+      {r:0, c:1},
+      {r:1, c:-1},
+      {r:1, c:0},
+      {r:1, c:1}];
     this.state = {
+      board : this.initBoard,
       currentWord: '',
+      currentWordPosition: [],
       wordList: {},
       totalScore : 0
     }
   }
+  // 1. click on the tile
+  // 2. update tile selected to true.
+  // 2.1 Can select and unselect the tile
+  // 2.2 Can only unselect the last tile
+  // 2.3 Update currentWord as we select and unselect
+  // 2.4. TODO: should be only able to select the surrounding cells
+  // 2.5 Make a copy of board, word, currentWordPositions, etc
+  // 2.6 Mutate the state
+  // 3. render the board with updated tile so it renders as active
+  handleClick(rowId, columnId) {
+    const newBoard = copyBoard(this.state.board);
+    const selected = newBoard[rowId][columnId].selected;
+    let currentWord = this.state.currentWord;
+    let newCurrentWordPosition = this.state.currentWordPosition.slice();
+    const lastLetter = newCurrentWordPosition[newCurrentWordPosition.length - 1]
 
-  //tile() {
-  //    const tile = {
-  //      letter: ,
-  //      isSelected: false,
-  //      position: {row: rowIndex, column: col}
-  //    }
-  //}
-  handleClick(letter) {
-    // if letter is within the surround previous letter
+    if(selected && rowId === lastLetter.rowId && columnId === lastLetter.columnId)
+    {
```

Comment:
Brace should be on the if statement line.  There should be a space after if.
Extract the if statements blocks to separate functions to add clarity.